docs/PERSONAS.md

```diff
@@ -4,11 +4,35 @@ In order to abstract away from specific use cases and
 implementation details, we will use terms from the story
 of *Glen and Larry’s Ice Cream Company*.
 
-Each of these personas describes those involved in the
+Each persona describes those involved in the
 ice cream supply chain, and has a relevant parallel
 relating to OCI and the surrounding software ecosystem.
 
-## 1. The Dairy Farmer 🐮
+In this document, we aim to characterize which persona(s)
+this working group is designing around.
+
+## Which persona are we designing around?
+
+From the list of below, we are primarily designing to improve
+the function of
+**(3) The Refrigerated Truck Driver**.
+
+This working group is equivalent to the corporate strategy
```

Comment:
As an OSS group, any comments of "corporate strategy" seem to set off flags for folks.
Is this more about adapting to newer standards?

Another shipping analogy. 
Shipping of goods has been around a long time. The standard was to load cargo, by hand with Stevedores

Enter the shipping container. It provided a standard way to ship a collection of things. 
- [How a Steel Box Changed the World: A Brief History of Shipping](https://www.youtube.com/watch?v=0MUkgDIQdcM&t=94s) and [A Brief History of the Shipping Container](https://www.maritime-executive.com/editorials/a-brief-history-of-the-shipping-container)

New ships know how to efficiently load, secure and unload these containers. 
You could still load these on "older ship", with no data-loss. 

Is this a newer truck metaphor?

Reply:
I've removed the word "corporate" here to address your first point.

If you want to point to other shipping analogies in this section of the doc (or history etc.), please submit a PR to add them. They're more useful in the doc than lost in a PR comment.

> New ships know how to efficiently load, secure and unload these containers.
You could still load these on "older ship", with no data-loss.

This concerns the upgrade path, correct? I actually address this point just a few sentences below:

> How do we ensure business continues to operate smoothly if not everyone adopts our new process on day 1?

I've also opened a PR for a separate doc where we can discuss the upgrade path in detail: https://github.com/opencontainers/wg-reference-types/pull/6